Remove functions deprecated in v0.3.0

allardbrain opened the pull request:
Fixes #483

Review by @ritzvik requested

### Changes
This PR removes the deprecated `plot_hypersurface()` and `show()` methods of the `SchwarzschildEmbedding` class in `src/einsteinpy/hypersurface/schwarzschildembedding.py`.

### Docs
1. This example in the docs may need to be deprecated, versioned or removed:
https://docs.einsteinpy.org/en/stable/examples/Plotting_spacial_hypersurface_embedding_for_schwarzschild_spacetime.html

2. I checked the Sphinx source files in `docs/source/api/hypersurface/` and neither file therein needed any changes as a result of this PR.

### Tests
- I ran the test suite with `python -c "import einsteinpy.testing; einsteinpy.testing.test()"`
- The test output is `231 passed, 6 xfailed in 170.59s (0:02:50)` for both the `master` branch I forked and my feature branch.

ritzvik commented:
> ### Docs
> 1. This example in the docs may need to be deprecated, versioned or removed:
>    https://docs.einsteinpy.org/en/stable/examples/Plotting_spacial_hypersurface_embedding_for_schwarzschild_spacetime.html
>

https://docs.einsteinpy.org/en/latest/examples/Plotting%20spacial%20hypersurface%20embedding%20for%20schwarzschild%20spacetime.html 
The docs have been updated. You are looking at the stable version instead of latest. No need to worry

ritzvik commented:
You may need to change your email `config` in git, and redo the commits if you want your commit to get linked with your github profile. 
https://www.git-tower.com/learn/git/faq/change-author-name-email

The email should be same as that used to sign up with GitHub. 
If that's not an issue for you, I would merge. Everything looks good.

Thanks @allardbrain

allardbrain commented:
Thanks for your quick reply! I will change my git user's email address and re-submit this tomorrow. Appreciate you calling that out.

ritzvik commented:
Because the `master` has moved ahead, you may need to rebase. https://github.com/einsteinpy/einsteinpy/wiki/How-to-refactor-commits

allardbrain commented:
Hi @ritzvik thanks for the tip about rebasing the `master` branch. I've changed my git email address (had a typo in it) and rebased the `master` branch. Please confirm these changes are OK. Thanks!

ritzvik commented:
@allardbrain , thanks for all. Merging.
